ckanext/ontario_theme/templates/internal/organization/read_base.html

```diff
@@ -1,28 +1,57 @@
 {% ckan_extends %}
 
+{% block primary %}
+  <div class="primary ministry-page col-xs-12" role="main">
+    {% block primary_content %}
+      <article class="module">
+        {% block page_header %}
+          <header class="page-header ministry-page-header">
+            <div class="content_action">
+              {% block content_action %}
+                {% if h.check_access('organization_update', {'id': c.group_dict.id}) %}
+                  {% link_for _('Edit'), named_route='organization.edit', id=c.group_dict.name, class_='btn btn-default btn-secondary', icon='wrench' %}
+                {% endif %}
+              {% endblock content_action %}
+            </div>
+            {% snippet 'snippets/organization.html', organization=group_dict, show_nums=true, group_type=group_type %}
+            <ul class="nav nav-tabs">
+              {# Override CKAN: Do not pass any icons to nav bar menu items #}
+              {% block content_primary_nav %}
+                {{ h.build_nav_icon(group_type + '.read', _('Datasets'), id=group_dict.name) }}
+                {{ h.build_nav_icon(group_type + '.activity', _('Activity Stream'), id=group_dict.name, offset=0) }}
+                {{ h.build_nav_icon(group_type + '.about', _('About'), id=group_dict.name) }}
+              {% endblock content_primary_nav %}
+            </ul>
+          </header>
+        {% endblock page_header %}
+
+        <div>
+          {% if self.page_primary_action() | trim %}
+            <div class="page_primary_action">
+              {% block page_primary_action %}
+              {% endblock page_primary_action %}
+            </div>
+          {% endif %}
+
+          {% block primary_content_inner %}
+          {% endblock primary_content_inner %}
+
+          {% block organization_facets %}
+          {% endblock organization_facets %}
+        </div>
+      </article>
+    {% endblock primary_content %}
+  </div>
+{% endblock primary %}
+
 {% block subtitle %}
   {{ h.get_translated(c.group_dict,"title") }} - {{ _('Ministries') }}
 {% endblock subtitle %}
 
 {% block breadcrumb_content %}
   <li>{% link_for _('Ministries'), named_route='organization.index', named_route=group_type + '_index' %}</li>
-  <li id="ministry-name"
-      ministry-name="{{ c.group_dict.title }}"
-      ministry-name-fr="{{ h.ontario_theme_get_translated_lang(c.group_dict, 'title', 'fr') }}"
-      class="active">
-    {% link_for h.get_translated(c.group_dict,"title")|truncate(35), named_route='organization.read', id=c.group_dict.name %}
-  </li>
+  <li id="ministry-name" ministry-name="{{ c.group_dict.title }}" ministry-name-fr="{{ h.ontario_theme_get_translated_lang(c.group_dict, 'title', 'fr') }}" class="active">{% link_for h.get_translated(c.group_dict,"title")|truncate(35), named_route='organization.read', id=c.group_dict.name %}</li>
 {% endblock breadcrumb_content %}
 
-{% block content_action %}
```

Comment:
What was all this code that you removed?

Reply:
It was relocated to the new primary block (top of file), which overrides the primary block in `page.html` & CKAN to allow for the layout change. The empty blocks in {% block primary %} are being filled by `read.html` in this folder.